client/src/components/Collections/common/CollectionEditView.vue

```diff
@@ -0,0 +1,260 @@
+<template>
+    <div>
+        <h4>{{ l("Edit Collection Attributes") }}</h4>
+        <b-alert show variant="warning" dismissible>
+            {{
+                l(
+                    "Collections are immutable. This means there will be some things you cannot change without creating a new collection. "
```

Comment:
```suggestion
                    "This will create a new Collection in your History. Your quota usage will not increase. "
```
I think that's less technical and conveys the same information.